src/pages/profile/ProfileSections/UserReviews.jsx

```diff
@@ -1,21 +1,20 @@
-import CommentCard from "../../../components/TypesOfRecipeCards/CommentCard";
+import ProfilePost from "../../../components/TypesOfRecipeCards/ProfilePost";
+
 const UserReviews = ({ user, reviews }) => {
   //Show only user's reviews
-  const hasReviews = reviews.length > 0;
+  const isEmptyReviews = reviews.length === 0 ? true : false;
```

Comment:
It's better to have positive names for variables. Currently it is negative, and you have !isEmptyReviews, which is double negative. This is harder to understand. 

You also don't need to add `? true : false`. This is redundant.